### DISCUSSION TITLE
Hide Export Button ???

### DISCUSSION THREAD

**Discussion by jbk68** [cortexrd/Knack-Toolkit-Library#150] [General]
I have added a field (show export button) in my customer's profile table that has the option of YES or NO.  I'd like to hide Knack's export button if the field value is "NO" on a certain table/scene for selected customers.  Is this possible using KTL ?

**Comment by cortexrd** [comment]
Jon,

FYI, if you do an right-click and Inspect on the Export button, it will lead you to the **.kn-export-button** class, under your view_id.

To hide the Export button, you can execute these lines in your console:
```
const VIEW_ID_WITH_EXPORT_BUTTON = 'view_xxx';
$('#' + VIEW_ID_WITH_EXPORT_BUTTON + ' .kn-export-button').addClass('ktlDisplayNone');
```
To show it again:
`$('#' + VIEW_ID_WITH_EXPORT_BUTTON + ' .kn-export-button').removeClass('ktlDisplayNone');`

Also, you can do this to remove it completely from the DOM:
`$('#' + VIEW_ID_WITH_EXPORT_BUTTON + ' .kn-export-button').remove();`

But then , you can't show it anymore.

Now, to answer your question, you will need the **_cls** keyword with the **ktlCond** option.   I am working on this now, and should release it later today.

Norm

**Reply by jbk68** [reply]
As always, thank you Norm for your response and I look forward to a solution using KTL.

Thanks again... JON

**Reply by amandajane-mo** [reply]
So excited about the introduction of ktlCond with _cls - that will be amazing

Amanda

**Comment by cortexrd** [comment]
Jon,

Did you have the chance to try the **_cls** keyword with the **ktlCond** option?  It's been released in **v0.18.8**.

In the grid where you want to hide the export button, add this keyword string:

`_cls=[ktlHidden], [ktlCond, is, NO, show export button, Account Details View]`

This will work if your page has an **Account Details View** that shows your field **show export button**.

Norm

**Reply by jbk68** [reply]
Good Morning Norm, I wasn't able to get the _cls keyword to work.  

Not sure if it matters or not, but my "show export button" field is on a connection table but visible on the grid/view I want to hide the export button if of value of field is "NO".
